config/application.yml.default

```diff
@@ -182,7 +182,8 @@ risc_notifications_active_job_enabled: 'false'
 risc_notifications_rate_limit_interval: '60'
 risc_notifications_rate_limit_max_requests: '1000'
 risc_notifications_rate_limit_overrides: '{"https://example.com/push":{"interval":120,"max_requests":10000}}'
-ruby_workers_enabled: 'true'
+ruby_workers_cron_enabled: 'true'
+ruby_workers_idv_enabled: 'true'
```

Comment:
does there need to be a `risc_notifications_active_job_enabled`?

Reply:
that already existed https://github.com/18F/identity-idp/pull/5333/files#diff-2f33ba3c11539151a98fde08277eac1b5e17e8ea15aa61b22a0fa9ce8c18f01e, and I figured its name matched these configs, so it wasn't worth renaming:
- risc_notifications_rate_limit_interval
- risc_notifications_rate_limit_max_requests
- risc_notifications_rate_limit_overrides

Reply:
Oh oops, got it, makes sense